docker/gcp_pudl_etl.sh

```diff
@@ -4,6 +4,16 @@
 
 set -x
 
+function setup_dagster_storage() {
+    SHORT_GITHUB_REF=$(git rev-parse --short $GITHUB_REF)
+    export DAGSTER_PG_DB="dagster-storage-$SHORT_GITHUB_REF"
+
+    # Start dagster-storage Cloud SQL instance
+    gcloud sql instances patch dagster-storage --activation-policy=ALWAYS
+    # Create database
+    gcloud sql databases create $DAGSTER_PG_DB --instance=dagster-storage 
```

Comment:
1. Does this wait for the database to spin up before returning?
2. Does create throw an error if the DB already exists? Do we expect the DB to persist data over runs, or do we want to drop everything between runs? I see we delete the database below, but what happens if we error out and get interrupted before we delete the database?

Reply:
1. I'll double check the database is created before returning.
2. I figured it was fine to delete the database even if the build fails because the dagster event logs aren't actually that helpful when debugging build failures.